pkg/config/config.go

```diff
@@ -28,7 +28,8 @@ type Config struct {
 	// AccessRequestURL is a Granted Approvals URL that users can visit
 	// to request access, in the event that we receive a ForbiddenException
 	// denying access to assume a particular role.
-	AccessRequestURL string `toml:",omitempty"`
+	AccessRequestURL   string `toml:",omitempty"`
+	ProfileRegistryURL string `toml:",omitempty"`
```

Comment:
We should change this to a type that can support multiple registries - we should also store the `ref` for each registry.

Reply:
This has been done in my current changes. 🎉

Reply:
`ref` implementation is still WIP.